module/modmanager/manager.go

```diff
@@ -152,6 +154,7 @@ type Manager struct {
 	// viamHomeDir is the absolute path to the viam home directory. Ex: /home/walle/.viam
 	// `viamHomeDir` may only be the empty string in testing
 	viamHomeDir string
+	packagesDir string
```

Comment:
let's add short comment explaining what this is for

Reply:
done in d4fd40356

this just comes from config -- high level question here, are there other ways to push down config? can it be stored in the Context object, because we pass that down anyway? it is useful for config to be 'global' (ish). recognize there are tradeoffs here though

Reply:
I'd rather not store that in the context, I don't think this is that bad, ultimately it's more discoverable and easier to reason about